README.md

```diff
@@ -1,24 +1,105 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Micro Reddit Database ![](https://img.shields.io/badge/Ruby-CC342D?style=for-the-badge&logo=ruby&logoColor=white) ![](https://img.shields.io/badge/Ruby_on_Rails-CC0000?style=for-the-badge&logo=ruby-on-rails&logoColor=white)
 
-Things you may want to cover:
+>This database is a project developed as part of the Microverse curriculum, to be more specific, is part of the Ruby on Rails module.
+It consists in a database created for a Micro Reddit app, no front end implement at this point, only the database models
 
-* Ruby version
+### Relational Model
 
-* System dependencies
+![Database Model](./database_model.png)
 
-* Configuration
 
-* Database creation
 
-* Database initialization
+### 👷🏻 Built using
+    - Ruby
+    - Ruby On Rails
+    - Rubocop
+    - SQLite3
+    - Github
+    - RubyMine
 
-* How to run the test suite
+## Getting Started
 
-* Services (job queues, cache servers, search engines, etc.)
+To get a local copy up and running follow these simple example steps.
 
-* Deployment instructions
+### ✅ Prerequisites
 
-* ...
+* Make sure that **Ruby Environment** is installed in your computer (try to run _ruby_ from your prompt), if not you can install it from [here.](https://www.ruby-lang.org/en/downloads/)
+
+* [Download](https://github.com/kensayo/micro-reddit/archive/refs/heads/development.zip) and unzip **or** [clone this repository](https://docs.github.com/es/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
+
+
+### 👨🏻‍🔧 Setup
+
+- Open your terminal and move to the root of the project, using ```cd``` command.
+- Then you must run ```$ rails db:migrate```, so you can work locally with the database.
+- Run ```$ rails db:create``` so you create the database
+- You must run ```$ bundle install```  
+- Start server using ```$ rails s```
+- Finally go to the Ruby console with `$ rails c`
```

Comment:
You should ask the user to use the command **rails bundle install** first then **rails db:create** after that **rails db:migrate** and finally **rails s** and/or **rails c**